README.md

```diff
@@ -336,6 +335,11 @@ These tools define a documenting syntax for CSS. You would e.g. write your compo
 
 [Source](https://github.com/dimanech/atlas-guide) | [Demo](https://dimanech.github.io/atlas-guide/) | **NodeJS, Mustache, Markdown, Static Site Generator**
 
+### [Sytleguide](https://hugeinc.github.io/styleguide/index.html)
```

Comment:
Typo: "Sytleguide" -> "Styleguide"

Also, I'd like to see this named "Huge inc. - Styleguide" or something.
Just "Styleguide" for a name is a bit generic ;-)

Reply:
😳 My bad! Good looking out! 👀 Thank you! ✌️